commons/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="fr.free.nrw.commons"
+    package="free.nrw.commons"
```

Comment:
You are right that the package names in the code does not need to be changed.
I meant searching for "free.nrw" without the "fr.", or old occurrences of "org.wikimedia.commons"
When everything should work but does not, Stackoverflow is a good place to ask, people there usually know immediately what the problem is or can point you to possible solutions, I ask questions there almost everyday :-)